daemons/dss-index/.chalice/config.json

```diff
@@ -11,5 +11,6 @@
   },
   "version": "2.0",
   "lambda_timeout": 300,
-  "lambda_memory_size": 1536
+  "lambda_memory_size": 1536,
+  "reserved_concurrent_executions": 400
```

Comment:
`reserved_concurrent_executions` may need to be scaled with the ES cluster size, which is different for prod.

Reply:
That makes sense and I would spend time optimizing the scale for both configs, but feel I think that this will work and is "good enough" for a lambda that we intend to deprecate soon.  Let me know though if you feel like it's worth investing more time!